raco/language/myrialang.py

```diff
@@ -632,7 +639,7 @@ def num_tuples(self):
     def compileme(self, inputid):
         if len(self.hash_columns) == 1:
             pf = {
-                "type": "SingleFieldHash",
+                "type": "RawValue" if self.is_raw_val else "SingleFieldHash",
```

Comment:
I added a new type of Shuffling to MyriaX - a shuffle that sends to the worker id matching a raw attribute value (i.e. the non-hashed value). RACO needs to be aware of this.